homeassistant/components/zwave_js/__init__.py

```diff
@@ -623,7 +624,7 @@ async def async_on_node_ready(self, node: ZwaveNode) -> None:
         )
 
         # add listeners to handle new values that get added later
-        for event in ("value added", "value updated", "metadata updated"):
+        for event in ("value added", EVENT_VALUE_UPDATED, "metadata updated"):
```

Comment:
I added this refactor cause it was hard to find the event callback that was at play.